Database/Corrections/sodQuestFixes.lua

```diff
@@ -279,6 +279,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.startedBy] = {{213077,214070,214096,214098,214099,214101,}},
             [questKeys.specialFlags] = specialFlags.REPEATABLE,
         },
+        [78909] = {
+            [questKeys.requiredSourceItems] = {211419},
```

Comment:
What is that for? This item is an objective so it should already show on the map 🤔

Reply:
You need to get the drop 211419 in order to be able to start the dialogue and pick up the quests (where the object is also an objective in the quest)
https://www.wowhead.com/classic/item=211419/handful-of-shifting-scales

Reply:
I see, thanks for pointing out.

Well that field is used for items you need to collect while on a quest. Not as a real objective but for certain steps.

Not sure what's the best case to handle this case. The "pre quest" is looting the item, while the item does not start the quest directly 🤔

@Cabro do you have an idea? Maybe a fake quest which we use as pre quest and mark is as exclusiveTo?

Reply:
This is exactly the case for TF bindings quest. There is a part in code where we look for "looted" strings. Maybe we can do another if case to check the id of the item we looted, and then just make the respective quest visible?

Reply:
> This is exactly the case for TF bindings quest. There is a part in code where we look for "looted" strings. Maybe we can do another if case to check the id of the item we looted, and then just make the respective quest visible?

There is? 🤔 Do you know where? I didn't find anything after a quick search.

Reply:
The check Cabro mentions should ideally be found in `function _QuestieQuest:DrawAvailableQuest(quest)` in QuestieQuest.lua or in `function QuestieDB.IsDoable(questId, debugPrint, returnText, returnBrief)` in QuestieDB.lua right?

Reply:
> > This is exactly the case for TF bindings quest. There is a part in code where we look for "looted" strings. Maybe we can do another if case to check the id of the item we looted, and then just make the respective quest visible?
> 
> There is? 🤔 Do you know where? I didn't find anything after a quick search.

QuestieTracker.lua:431

Reply:
Okay so here is my idea:

For now lets just update the objective text for that quest to explain that you need to Scale in order to accept the quest.

I created #5385 to keep track of a better idea.